mqtt-discovery.sh

```diff
@@ -721,15 +748,15 @@ delete_legacies() {
 delete_legacies_singles() {
   vin=$1
 
-  log_notice "Deleting single MQTT entities topics"
-  eval $MOSQUITTO_PUB_BASE -t homeassistant/button/tesla_ble/climate-off/config -n
+  log_notice "delete_legacies_singles; deleting legacy single MQTT entities topics"
   eval $MOSQUITTO_PUB_BASE -t homeassistant/button/tesla_ble/climate-on/config -n
-  eval $MOSQUITTO_PUB_BASE -t homeassistant/button/tesla_ble/trunk-close/config -n
+  eval $MOSQUITTO_PUB_BASE -t homeassistant/button/tesla_ble/climate-off/config -n
```

Comment:
Don't we need to set the vin in the topic name?

Reply:
Oh this is bad... I was wondering why I have topis named tesla_ble with no names in MQTT (same comment below).